lambdas/api/put_dmps_uploads/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import { lambdaRequestTracker } from 'pino-lambda';
```

Comment:
Do we need to add the endpoint here? https://github.com/CDLUC3/dmsp_api_prototype/wiki/api-overview

Reply:
💯